File: scripts/algorithms/M/Minimum Cost to Move Chips to The Same Position/Minimum Cost to Move Chips to The Same Position.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        count_even=0
        count_odd=0
        for i in position:
            if i%2==0:
                count_even+=1
            else:
                count_odd+=1
        return min(count_even,count_odd)

File: scripts/algorithms/M/Minimum Cost to Move Chips to The Same Position/Minimum Cost to Move Chips to The Same Position.rs
// Runtime: 0 ms (Top 100.0%) | Memory: 2.20 MB (Top 20.0%)

impl Solution {
    pub fn min_cost_to_move_chips(position: Vec<i32>) -> i32 {
        let odd = position.iter().filter(|&p| p % 2 == 1).count();
        std::cmp::min(odd, position.len() - odd) as i32
    }
}